cucumber/authenticators_jwt/features/authn_jwt.feature

```diff
@@ -130,3 +130,58 @@ Feature: JWT Authenticator - JWKs Basic sanity
     """
     CONJ00100E Annotation, 'custom-claim', is empty
     """
+
+  Scenario: Ignore invalid annotations
+    Given I have a "variable" resource called "test-variable"
+    Given I extend the policy with:
+    """
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw: invalid
+        authn-jwt: invalid
+        authn-jwt/raw/sub/sub: invalid
+        authn-jwt/raw2/sub: invalid
+        authn-jwt/raw/sub: valid
+        authn-jwt/raw/namespace-id: valid
+        authn-jwt/raw/project-path: valid
```

Comment:
If one of the valid message will fail , the test will not fail because we dont verify each line specific, and since we have valid fields it will succeed.  right ? if so we need to handle this case

Reply:
I think youre right, the test will succeed
what to you mean by "handle this case"?

Reply:
maybe test separately each one of them ? If so , left here only one valid message since all the rest not really verified

Reply:
ill correct myself, each of the annotations is checked through logs, invalid annotation wont log `CONJ00048D Validating resource restriction on request:` so the test will fail